### PR TITLE
Bump snaps packages

### PR DESCRIPTION
**Pull request by FrederikBolding**
## **Description**

Bump snaps packages to latest.

Summary of changes in the snaps deps:
- Add support for unencrypted state
- Add `onHomePage` export and endowment
- Add manifest localization support
- Fix some issues with stream communication

### CODE REVIEW

**Review by bowensanders — COMMENTED** [review]
LGTM

**Comment by bowensanders** [comment]
QA Passes ✅

File: shared/constants/snaps/permissions.ts [hunk 1]
```diff
@@ -26,6 +26,8 @@ export const ExcludedSnapPermissions = Object.freeze({
 });
 
 export const ExcludedSnapEndowments = Object.freeze({
+  'endowment:page-home':
+    'This permission is still in development and therefore not available.',
   ///: BEGIN:ONLY_INCLUDE_IN(build-main)
   'endowment:lifecycle-hooks':
```
Comment (Mrtenz) [inline shared/constants/snaps/permissions.ts:32]: Should we enable this?

**Review by Mrtenz — APPROVED** [review]

File: shared/constants/snaps/permissions.ts (hunk 1 above)
Comment (FrederikBolding) [reply shared/constants/snaps/permissions.ts:32]: Will enable in a follow up PR. The follow up PR will add the permission copy and the actual feature integration.

**Review by GuillaumeRx — APPROVED** [review]

**Pull request merged**